config/jobs/kubernetes/sig-node/node-kubelet.yaml

```diff
@@ -54,7 +54,7 @@ periodics:
   annotations:
     fork-per-release: "true"
     fork-per-release-periodic-interval: 1h 2h 6h 24h
-    testgrid-dashboards: sig-release-master-blocking, sig-node-release-blocking
+    testgrid-dashboards: sig-node-release-blocking
```

Comment:
It could be added to `sig-node-critical` tab rather than release blocking one
https://testgrid.k8s.io/sig-node-critical

Reply:
A possible follow up then? @Namanl2001